cupyx/tools/install_library.py

```diff
@@ -124,38 +124,21 @@ def _make_cutensor_record(
     }
 
 
-_cutensor_records.append(_make_cutensor_record(
-    '11.6', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.5', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.4', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.3', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.2', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.1', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '11.0', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
-_cutensor_records.append(_make_cutensor_record(
-    '10.2', '1.4.0',
-    'libcutensor-linux-x86_64-1.4.0.6-archive.tar.xz',
-    'libcutensor-windows-x86_64-1.4.0.6-archive.zip'))
+def _make_cutensor_record(cuda_version):
```

Comment:
cuTENSOR provides the same headers and binaries for each CTK version it supports these days, so I wrapped up the calls to make cutensor records.